shared/api/storage_pool_volume.go

```diff
@@ -8,13 +8,19 @@ type StorageVolumesPost struct {
 
 	Name string `json:"name" yaml:"name"`
 	Type string `json:"type" yaml:"type"`
+
+	// API extension: storage_api_local_volume_handling
+	Source StorageVolumeSource `json:"source" yaml:"source"`
 }
 
 // StorageVolumePost represents the fields required to rename a LXD storage pool volume
 //
 // API extension: storage_api_volume_rename
 type StorageVolumePost struct {
 	Name string `json:"name" yaml:"name"`
+
+	// API extension: storage_api_local_volume_handling
+	Source StorageVolumeSource `json:"source" yaml:"source"`
```

Comment:
Hmm, this looks wrong. What I'd expect to see here is:

```go
// API extension: storage_api_local_volume_handling
Pool string `json:"pool" yaml:"pool"`
```

So if I want to rename a volume within the same pool, I POST to it with the new Name set and if I want it moved from one pool to another, I set the Pool property to the target pool.